ra-optimisation/linear-rao/src/test/java/com/farao_community/farao/linear_rao/config/LinearRaoParametersTest.java

```diff
@@ -61,4 +61,10 @@ public void setPstSensitivityThreshold() {
         raoParameters.setPstSensitivityThreshold(5.0);
         assertEquals(5.0, raoParameters.getPstSensitivityThreshold(), DOUBLE_TOLERANCE);
     }
+
+    @Test
+    public void setPstPenaltyCost() {
+        raoParameters.setPstPenaltyCost(0.5);
```

Comment:
Try the set method with a value different than default one

Reply:
now default value is different :smile: